### PR TITLE
Ccp 1357: Issue adding variant products on GMD

### PR DESCRIPTION
**Pull request by DannyShopgate**
## Description
PWA v6.X requires use of conditioner for variant products on GMD and had to update to use Theme Context - Product Context

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

**Comment by DannyShopgate** [comment]
Used shop number 31369 example variant product: Trixie Trixie Tennisbälle - 48750, example grouped product: 	Almo Nature HFC Light Goldbrasse & Kartoffeln 70g - 46345

**Comment by DannyShopgate** [comment]
Created https://jira.shopgate.guru/browse/TD-887 to update unit test coverage.

File: frontend/components/GroupedProducts/components/Product/index.jsx [hunk 1]
```diff
@@ -80,6 +86,7 @@ const Product = ({
                     noShadow: renderFlatButtons,
                     type: ADD_TO_CART_BUTTON_TYPE_GROUPED,
                   }}
+                  conditioner={conditioner}
```
Comment (sznowicki) [inline frontend/components/GroupedProducts/components/Product/index.jsx]: Please add conditioner to the test of this component so it doesn't render warning.

Comment (DannyShopgate) [reply frontend/components/GroupedProducts/components/Product/index.jsx]: Ah right. Updated.

**Pull request merged**